kubeflow/Makefile

```diff
@@ -28,28 +30,13 @@ BUILD_DIR=.build
 # TODO(jlewi): Should we make this a setter to make it easier to update programmatically?
 MANIFESTS_URL=https://github.com/kubeflow/manifests.git@master
 
-#***********************************************************************************************************************
-# Edit this section to set the values specific to your deployment
-
-.PHONY: set-values
-set-values:
-	kpt cfg set ./instance gke.private false
-
-	kpt cfg set ./instance mgmt-ctxt <YOUR_MANAGEMENT_CTXT>
-
-	kpt cfg set ./upstream/manifests/gcp name <YOUR_KF_NAME>
-	kpt cfg set ./upstream/manifests/gcp gcloud.core.project <PROJECT_TO_DEPLOY_IN>
-	kpt cfg set ./upstream/manifests/gcp gcloud.compute.zone <ZONE>
-	kpt cfg set ./upstream/manifests/gcp location <REGION OR ZONE>
-	kpt cfg set ./upstream/manifests/gcp log-firewalls false
-
-	kpt cfg set ./upstream/manifests/stacks/gcp name <YOUR_KF_NAME>
-	kpt cfg set ./upstream/manifests/stacks/gcp gcloud.core.project <PROJECT_TO_DEPLOY_IN>
-
-	kpt cfg set ./instance name <YOUR_KF_NAME>
-	kpt cfg set ./instance location <YOUR_REGION or ZONE>
-	kpt cfg set ./instance gcloud.core.project <YOUR PROJECT>
-	kpt cfg set ./instance email <YOUR_EMAIL_ADDRESS>
+# Validate cluster values are changed from default dummy values
+.PHONY: validate-values
+validate-values:
+	@if [[ $(MGMTCTXT) == "MANAGEMENT-CTXT" || $(NAME) == "KUBEFLOW-NAME" || $(LOCATION) == "LOCATION" || $(ZONE) == "ZONE" || $(PROJECT) == "PROJECT" ]]; then\
```

Comment:
You have LOCATION here, but you didn't init its value

Reply:
Yes, it was present in the previous PR. Now available.